packages/core/ios/LottieReactNative/ContainerView.swift

```diff
@@ -295,6 +309,10 @@ class ContainerView: RCTView {
 
         applyColorProperties()
         playIfNeeded()
+
+        animationView?.animationLoaded = { animationView, animation in
```

Comment:
```suggestion
        animationView?.animationLoaded = { [weak self] animationView, animation in
```

Reply:
To follow up, after the weak self, use something like:
```swift
guard let self = self else { return }
```
To keep in line with the way we handle reference cycles. It also let's you keep the rest of your code as is